exporter/loadbalancingexporter/resolver_static.go

```diff
@@ -53,7 +53,9 @@ func newStaticResolver(endpoints []string) (*staticResolver, error) {
 }
 
 func (r *staticResolver) start(ctx context.Context) error {
-	r.resolve(ctx) // right now, this can't fail
+	if _, err := r.resolve(ctx); err != nil {  // right now, this can't fail
+		log.Fatalln("error while running resolve")
```

Comment:
I think we would like to learn more about the error:

```suggestion
		log.Fatalf("error while running resolve: %v", err)
```

Also, not fully sure if this error should be fatal or e.g. disable the output of the exporter, though I think this counts as [startup](https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#do-not-crash-after-startup) and should be ok

Reply:
Hmm, `start` can return error, disregard my comment,  this function should just return the error